explorer/src/lib/components/blocks-table/BlocksTable.svelte

```diff
@@ -31,9 +31,9 @@
   <TableHead>
     <TableRow>
       <TableCell type="th"># Block</TableCell>
-      <TableCell type="th">Fee</TableCell>
+      <TableCell type="th">Fee (Dusk)</TableCell>
       <TableCell type="th">Txn(s)</TableCell>
-      <TableCell type="th">Rewards</TableCell>
+      <TableCell type="th">Rewards (Dusk)</TableCell>
```

Comment:
I noticed this isn't in the Figma design @laremas

Reply:
Same as above